README.md

```diff
@@ -1,127 +1,127 @@
 
 ### Deploy from the Azure Portal
-
+<!-- markdownlint-disable MD013 -->
 1. Deploy Mission Landing Zone into `AzureCloud` or `AzureUsGovernment` from the Azure Portal:
 
-    <!-- markdownlint-disable MD013 -->
-    <!-- allow for longer lines to acommodate button links -->
     | Azure Commercial | Azure Government |
     | :--- | :--- |
     | [![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fmissionlz%2Fmain%2Fsrc%2Fbicep%2Fmlz.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fmissionlz%2Fmain%2Fsrc%2Fbicep%2Fform%2Fmlz.portal.json) | [![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fmissionlz%2Fmain%2Fsrc%2Fbicep%2Fmlz.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fmissionlz%2Fmain%2Fsrc%2Fbicep%2Fform%2Fmlz.portal.json) |
-    <!-- markdownlint-enable MD013 -->
-
-1. After a successful deployment, see our [examples](./src/bicep/examples/README.md) directory for how to extend the capabilities of Mission Landing Zone.
-
-    - Extend your Mission Landing Zone deployment with the templates at [src/bicep/examples](./src/bicep/examples/README.md)
+<!-- markdownlint-enable MD013 -->
 
-### Deploy using Azure CLI
+2. After a successful deployment, see our [examples](./src/bicep/examples/README.md) directory for how to extend the capabilities of Mission Landing Zone.
 
-Or, you can deploy Mission Landing Zone using Azure CLI.
+### Deploy using the Azure CLI
 
-Don't have Azure CLI? Here's how to get started with Azure Cloud Shell in your browser: <https://docs.microsoft.com/en-us/azure/cloud-shell/overview>
-
-1. Clone the repository:
+1. Clone the repository and change directory to the root of the repository:
 
     ```plaintext
     git clone https://github.com/Azure/missionlz.git
+    cd missionlz
     ```
 
-1. Deploy Mission Landing Zone with the `az deployment sub create` command:
+1. Deploy Mission Landing Zone with the [`az deployment sub create`](https://docs.microsoft.com/en-us/cli/azure/deployment/sub?view=azure-cli-latest#az_deployment_sub_create) command. For a quickstart test deployment into the current AZ CLI subscription we suggest setting these parameters:
 
-    ```plaintext
-    az deployment sub create \
-      --name myMlzDeployment \
-      --location eastus \
-      --template-file ./missionlz/src/bicep/mlz.bicep
-    ```
+    - `--name`: (optional) The deployment name, which is visible in the Azure Portal under Subscription/Deployments.
+    - `--location`: (required) The location to store the deployment metadata.
```

Comment:
This argument list references local locations and cloud locations. I thought this might be helpful so as to clarify this is not a location on disk to store deployment metadata?

```suggestion
    - `--location`: (required) The Azure region to store the deployment metadata.
```